test/protractor/pages/app.js

```diff
@@ -7,7 +7,8 @@ module.exports = api;
 api.login = function(identity) {
   element(by.buttonText('Sign In')).click();
   const c = $('br-authn-password');
-  c.element(by.brModel('$ctrl.sysIdentifier')).sendKeys(identity.sysIdentifier);
+  c.element(by.brModel('$ctrl.sysIdentifier'))
+    .sendKeys(`${identity.sysIdentifier}`);
```

Comment:
should be no need for wrapping this in backticks, so just should suffice.
```
.sendKeys(identity.sysIdentifier);
```

Reply:
wondered about that, ok

Reply:
@jeiholzer Here's a spot where using a template would be useful: https://github.com/digitalbazaar/bedrock-angular-agreement/pull/10/files#diff-671e631aeda5fd27949e7ff95c8c4c11R16

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals